moveit_core/robot_model/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
+cmake_minimum_required(VERSION 3.5)
 set(MOVEIT_LIB_NAME moveit_robot_model)
 
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
```

Comment:
This is broken because of the standard specification in the top-level `CMakeLists.txt`
Please find my other comment on this for details.

Reply:
Thanks @v4hn, please see if https://github.com/ros-planning/moveit2/commit/18edf88b3e85f2c377e0618b3105ac55d0a5ee1a addresses your concerns